src/supplemental/nanolib/parquet/parquet_file_manager.cc

```diff
@@ -4,8 +4,8 @@
 void
 parquet_file_manager::add_queue(conf_exchange_node *node)
 {
-	file_queue_map[node->name] = make_shared<parquet_file_queue>(node->parquet);
-	file_queue_map[node->name]->init();
+	file_queue_map[node->topic] = make_shared<parquet_file_queue>(node->parquet);
```

Comment:
Need a urgent release?

Reply:
If config exchange name is the same with topic, it's OK and normaly it's same.